recipes/libcurl/all/conanfile.py

```diff
@@ -285,9 +304,53 @@ def _configure_autotools_vars(self):
 
             del autotools_vars['LIBS']
             self.output.info("Autotools env vars: " + repr(autotools_vars))
+
+        if tools.cross_building(self.settings):
+            if self.settings.os == "iOS": 
+                ios_dev_target = str(self.settings.os.version).split(".")[0]
+                if self.settings.arch in ["x86", "x86_64"]:
+                    iphoneos = "iphonesimulator"
+                    autotools_vars['CPPFLAGS'] = "-D__IPHONE_OS_VERSION_MIN_REQUIRED={}0000".format(ios_dev_target)
+                elif self.settings.arch in ["armv7", "armv7s", "armv8"]:
+                    iphoneos = "iphoneos"
+                    autotools_vars['CPPFLAGS'] = ""
+                else:
+                    raise ConanInvalidConfiguration("Unsuported iOS arch {}".format(self.settings.arch)) 
+
+                cc = tools.XCRun(self.settings, iphoneos).cc
+                sysroot = "-isysroot {}".format(tools.XCRun(self.settings, iphoneos).sdk_path)
```

Comment:
there is a helper method to get sysroot flag

Reply:
what do you mean? I thought I use it ?
`tools.XCRun(self.settings, iphoneos).sdk_path`